pkgs/by-name/ne/neverest/package.nix

```diff
@@ -34,7 +34,43 @@ rustPlatform.buildRustPackage rec {
     ++ lib.optional (installManPages || installShellCompletions) installShellFiles;
 
   buildInputs = [ ]
-    ++ lib.optionals stdenv.isDarwin (with darwin.apple_sdk.frameworks; [ AppKit Cocoa Security ])
+  ++ lib.optionals stdenv.isDarwin (with darwin.apple_sdk_11_0.frameworks; [
+  AppKit
+  ApplicationServices
+  AudioToolbox
+  AudioUnit
+  CFNetwork
+  Carbon
+  CloudKit
+  Cocoa
+  ColorSync
+  Combine
+  CoreAudio
+  CoreAudioTypes
+  CoreData
+  CoreFoundation
+  CoreGraphics
+  CoreImage
+  CoreLocation
+  CoreMIDI
+  CoreServices
+  CoreText
+  CoreVideo
+  CoreVideo
+  DiskArbitration
+  Foundation
+  IOKit
+  IOSurface
+  ImageIO
+  Metal
+  NetFS
+  OpenCL
+  OpenGL
+  QuartzCore
+  Security
+  ServiceManagement
+  SystemConfiguration
+  ])
```

Comment:
How come do we need all those frameworks? Is there any other Nix package with such definition?

Reply:
I don't have a Mac either, I'm just trying to work off of the hydra/CI logs and see if the CI passes (it does not, not sure why). If I can't get it to work I'll close this.